.github/workflows/build.yml

```diff
@@ -274,6 +274,33 @@ jobs:
             ${{ github.workspace }}/packages/**/*.tgz
             ${{ github.workspace }}/packages/serverless/dist-awslambda-layer/*.zip
 
+  job_playwright_test:
+    name: Playwright E2E Tests
```

Comment:
The name shouldn't be about the technology we are using, but about what the job does. We aren't testing playwright, but testing E2E (the browser SDK). For example, the job to run tests is "job_unit_test" and not "job_jest_test" or similar.

Reply:
Updating this as `job_e2e_browser_test`, the next step is to generalize this structure to all client-side SDKs, so it might be splitted further unless a single entry point is preferred.